CHANGELOG.md

```diff
@@ -136,6 +443,8 @@ Generated on 2022-08-20
 ### PRs
 |||
 |:---|:---|
+|[#6421](https://github.com/NVIDIA/spark-rapids/pull/6421)|Fix notebook and getting started examples [skip ci]|
```

Comment:
Why this PR is included in 22.08?

Reply:
![image](https://user-images.githubusercontent.com/8086184/195232133-2dbe287f-f737-4766-a7fc-d217ffc23b7a.png)

it was marked as 22.08 PR